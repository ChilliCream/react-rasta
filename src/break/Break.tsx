import "../utils/bootstrap";
import { ClassAttributes, HTMLAttributes } from "react";
import { StyledComponentClass } from "styled-components";
import { styled, Theme } from "../theme";

const Break = styled.div`
  width: 100%;
  height: 0;
`;

export default Break;
