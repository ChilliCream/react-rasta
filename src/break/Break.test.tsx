import {shallow} from "enzyme";
import "jest-styled-components";
import React from "react";
import Break from "./Break";

describe("<Break />", () => {
  it("should match the snapshot", () => {
    // act
    const result = shallow(<Break />);

    // assert
    expect(result).toMatchSnapshot();
  });
});
