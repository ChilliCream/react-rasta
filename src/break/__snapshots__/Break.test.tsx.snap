// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Break /> should match the snapshot 1`] = `
.c0 {
  width: 100%;
  height: 0;
}

<div
  className="c0"
/>
`;
